storage/migrations/02_runtimes.up.sql

```diff
@@ -70,6 +70,7 @@ CREATE TABLE chain.runtime_transactions
 );
 CREATE INDEX ix_runtime_transactions_tx_hash ON chain.runtime_transactions USING hash (tx_hash);
 CREATE INDEX ix_runtime_transactions_tx_eth_hash ON chain.runtime_transactions USING hash (tx_eth_hash);
+CREATE INDEX ix_runtime_transactions_timestamp ON chain.runtime_transactions (runtime, timestamp);
```

Comment:
Could avoid adding this index if `RuntimeTransactions` query is updated to also `LEFT JOIN on chain.runtime_blocks` and use the timestamp from there, since that one already has an index.

Reply:
That's one scary query already, let's not add more JOINs. Even as-is, it seems somewhat likely that it's possible to come up with a very resource-intensive query by combining just the right filters.

Reply:
:see_no_evil: